Rakefile

```diff
@@ -17,7 +17,7 @@
 
 require "yast/rake"
 
-Yast::Tasks.submit_to :sle15sp3
+Yast::Tasks.submit_to :sle15
```

Comment:
I would say this change is wrong. If you have the right version of yast-rake, then [sle15sp3](https://github.com/yast/yast-rake/blob/SLE-15-SP3/data/targets.yml#L83-L87) should work.